Update/toki_005.txt

```diff
@@ -2886,7 +2886,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#a59da9>大石</color>", NULL, "<color=#a59da9>Ooishi</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 11, "ps3/s11/11/420300130", 256, TRUE);
 	OutputLine(NULL, "「ですが、その意思は普通に生活を送っている人を死に追いやったり、あるいは殺しに駆り立てたりすることができる。",
-			NULL, "\"However, that will can drive normal people living otherwise peaceful lives to commit murder.", Line_WaitForInput);
+			NULL, "\"However, that will can drive normal people living otherwise peaceful lives to their deaths or to commit murder.", Line_WaitForInput);
```

Comment:
```suggestion
			NULL, "\"However, that will can drive normal people living otherwise peaceful lives to their deaths, or to commit murder.", Line_WaitForInput);
```